### PR TITLE
Readme update 2023

### PR DESCRIPTION
**Pull request by davidmirror-ops**
## Why are the changes needed?
Looking to make the README a more educational resource

## Check all the applicable boxes 

- [ ] I updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] All commits are signed-off.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,79 +30,136 @@ Flyte is an **open-source orchestrator** that facilitates building production-gr
...
+Flyte is an open-source orchestrator that facilitates building production-grade data and ML pipelines. It is built for scalability and reproducibility, leveraging Kubernetes as its underlying platform. With Flyte, user teams can construct pipelines using the Python SDK, and seamlessly deploy them on both cloud and on-premises environments, enabling distributed processing and efficient resource utilization.
+
+<h2 align="center">
+  Build
+</h2>
+<p>
+Write code in Python or any other language and leverage a robust type engine. 
+</p>
+
+<img alt="Getting started with Flyte" src="https://raw.githubusercontent.com/flyteorg/static-resources/main/common/flytereadmebuildv2.gif" style="width: 100%; height: auto;" />
```
Comment (samhita-alla) [inline README.md:42]: can we reduce the size of the GIF window?

Comment (samhita-alla) [reply README.md:42]: also, i'm not sure if this is a big ask, but could we consider including a different example here? :)
[this](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/basics/workflow.html) would be amazing.

File: README.md [hunk 2]
```diff
@@ -37,79 +30,136 @@ Flyte is an **open-source orchestrator** that facilitates building production-gr
...
+Go to the [Deployment guide](https://docs.flyte.org/en/latest/deployment/deployment/index.html) for instructions to install Flyte on different environments
 
-## Adopters
+## Tutorials
 
-Join the likes of LinkedIn, Spotify, Freenome, Pachama, Gojek, and Woven Planet in adopting Flyte. For a full list of adopters and information on how to add your organization or project, please visit our [ADOPTERS](https://github.com/flyteorg/community/blob/main/ADOPTERS.md) page.
 
+- [Fine-tune Code Llama on the Flyte codebase](https://github.com/unionai-oss/llm-fine-tuning/tree/main/flyte_llama#readme)
+- [Forecast sales with Horovod and Spark](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/forecasting_sales/index.html)
+- [Nucleotide Sequence Querying with BLASTX](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/blast/index.html)
```
Comment (samhita-alla) [inline README.md:113]: haven't tested these examples recently. but i think that's okay.

File: README.md [hunk 3]
```diff
@@ -37,79 +30,136 @@ Flyte is an **open-source orchestrator** that facilitates building production-gr
...
-:hourglass: **Timeline view**: Evaluate the duration of each of your Flyte tasks and identify potential bottlenecks. <br />
-:fast_forward: **GPU acceleration**: Enable and control your tasks’ GPU demands by requesting resources in the task decorator. <br />
-:whale: **Dependency isolation via containers**: Maintain separate sets of dependencies for your tasks so no dependency conflicts arise. <br />
-:arrows_counterclockwise: **Parallelism**: Flyte tasks are inherently parallel to optimize resource consumption and improve performance. <br />
-:floppy_disk: **Allocate resources dynamically** at the task level. <br />
-:play_or_pause_button: [Wait](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/advanced_composition/waiting_for_external_inputs.html) for **external inputs** before proceeding with the execution. <br />
+<img alt="Getting started with Flyte, showing the welcome screen and Flyte dashboard" src="https://raw.githubusercontent.com/flyteorg/static-resources/main/flytesnacks/getting_started/getting_started_console.gif" style="width: 100%; height: auto;" />
```
Comment (samhita-alla) [inline README.md:88]: we should probably place this GIF below `pyflyte run --remote hello_world.py hello_world_wf` command.

File: README.md [hunk 4]
```diff
@@ -37,79 +30,136 @@ Flyte is an **open-source orchestrator** that facilitates building production-gr
...
+📅 **Scheduling**: [Schedule](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/productionizing/lp_schedules.html) your data and ML workflows to run at a specific time. <br />
+📢 **Notifications**: Stay informed about changes to your workflow's state by configuring [notifications](https://docs.flyte.org/projects/cookbook/en/latest/auto_examples/productionizing/lp_notifications.html) through Slack, PagerDuty or email. <br />
+⌛️ **Timeline view**: Evaluate the duration of each of your Flyte tasks and identify potential bottlenecks. <br />
+💨 **GPU acceleration**: Enable and control your tasks’ GPU demands by requesting resources in the task decorator. <br />
+🐳 **Dependency isolation via containers**: Maintain separate sets of dependencies for your tasks so no dependency conflicts arise. <br />
+🔀 **Parallelism**: Flyte tasks are inherently parallel to optimize resource consumption and improve performance. <br />
+💾 **Allocate resources dynamically** at the task level. <br />
+
+
+## Who's using Flyte?
```
Comment (samhita-alla) [inline README.md:146]: is a question mark in this and the next heading a strict requirement? not having one will maintain consistency.

File: README.md (hunk 1 above)
Comment (davidmirror-ops) [reply README.md:42]: That's a good example yes, the problem is that the GIF will be a bit longer (more code) and this one doesn't showcase the subworkfows that are probably not the most prominent feature but something a bit beyond the hello world

**Review by eapolinario — APPROVED** [review]
This is great!

**Pull request merged**